res/controllers/Pioneer DDJ-200.midi.xml

```diff
@@ -0,0 +1,479 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset schemaVersion="1" mixxxVersion="2.0.0+">
+    <info>
+        <name>Pioneer DDJ-200</name>
+        <author>Dan Giddins</author>
+        <description>Pioneer DDJ-200 configuration for 2 decks.</description>
+    </info>
+    <controller id="DDJ-200">
+        <scriptfiles>
+            <file filename="Pioneer-DDJ-200-scripts.js" functionprefix="DDJ200"/>
+        </scriptfiles>
+        <controls>
+            <control>
+                <group>[EqualizerRack1_[Channel1]_Effect1]</group>
+                <key>parameter3</key>
+                <status>0xB0</status>
+                <midino>0x07</midino>
+                <options>
+                    <fourteen-bit-msb/>
+                </options>
+            </control>
```

Comment:
If it's not too much work, you could add `<description>` tags for each control that contain the button label on the controller. That would make it easy to see which button is mapped to what function.

Reply:
Done :)